content/webapp/pages/index.tsx

```diff
@@ -96,7 +94,12 @@ export const getServerSideProps: GetServerSideProps<
 
   const client = createClient(context);
 
-  const articlesQueryPromise = fetchArticles(client, { pageSize: 4 });
+  const articlesResponsePromise = getArticles({
+    params: {},
```

Comment:
This is interesting. I think it was made required because in Search we want to know the sort order at a minimum, but here we rely on the default? Is the default last published?
Should it be a required prop? 🤔

Reply:
> Is the default last published

I confess I didn't research this beyond looking at what was returned and assuming that is what's happening. I'm not sure if it should be required (I personally don't mind that it is)

Reply:
As long as the behaviour is the same-ish, with the added bonus of being able to control sorting better (taking into account `overrideDate`, finally!)